<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google Music Web player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>&amp;Player</string>
    </property>
    <addaction name="actionPlayPause"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrev"/>
    <addaction name="actionStatus"/>
   </widget>
   <addaction name="menuPlayer"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPlayPause">
   <property name="text">
    <string>&amp;Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>P&amp;rev</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
